external/diagnostics_utils/diagnostics_utils/registry.py

```diff
@@ -0,0 +1,53 @@
+from collections import defaultdict
```

Comment:
Same as the prognostic run diags registry.

Reply:
Then should we avoid copy paste?